### ISSUE TITLE
Broken import for EmptyResultSet on Django 3.1

### ISSUE THREAD

**Issue by rvinzent** [M1hacka/django-pg-returning#19]
Installing the package and importing the `UpdateReturningManager` is broken on Django 3.1

It raises
```
cannot import name 'EmptyResultSet' from 'django.db.models.query' (/Users/rvinzent/invitae/lab-entity-service/env/lib/python3.8/site-packages/django/db/models/query.py)
```

`EmptyResultSet` appears to have moved to `django.core.exceptions` in Django 3.1

**Comment by M1ha-Shvn** [comment]
Thanks, I'll try to fix this compatibility issue soon

**Comment by M1ha-Shvn** [comment]
Released [version 1.2.5](https://github.com/M1ha-Shvn/django-pg-returning/releases/tag/v1.2.5)
